provider/aws/formation/rack.json

```diff
@@ -2943,6 +2943,7 @@
                 "iam:CreatePolicyVersion",
                 "iam:CreateRole",
                 "iam:CreateUser",
+                "iam:DeleteAccessKey",
```

Comment:
is this intended?

Reply:
yes, while testing I found an issue on deleting the rack resources. The iam permission is needed to delete the resources.